packages/devtools_app/lib/src/connect_screen.dart

```diff
@@ -115,7 +115,6 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
       );
       return;
     }
-
```

Comment:
nit: revert this probably accidental change.

Reply:
reverted.